src/linux/simulate.rs

```diff
@@ -1,79 +1,85 @@
 use crate::linux::keycodes::code_from_key;
 use crate::rdev::{Button, EventType, SimulateError};
+use std::convert::TryInto;
+use std::os::raw::c_int;
 use std::ptr::null;
 use x11::xlib;
 use x11::xtest;
-static TRUE: i32 = 1;
-static FALSE: i32 = 0;
 
-unsafe fn send_native(event_type: &EventType, display: *mut xlib::Display) -> Result<(), ()> {
-    match event_type {
+static TRUE: c_int = 1;
+static FALSE: c_int = 0;
```

Comment:
I my other PR, I moved common definition of TRUE and FALSE into src/linux/common.rs which tries to limit repetitions.

Reply:
I guess I need to rebase again to use those.

Reply:
No I will do it, I'm adding grab functionnality, it's going to take some time.

God x11 is messy...